src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/pullrequest/BitbucketPullRequestValueDestination.java

```diff
@@ -44,14 +45,14 @@
     @JsonCreator
     public BitbucketPullRequestValueDestination(@NonNull @JsonProperty("repository") BitbucketCloudRepository repository,
                                                 @JsonProperty("branch") BitbucketCloudBranch branch,
-                                                @NonNull @JsonProperty("commit") BitbucketCloudCommit commit) {
+                                                @JsonProperty("commit") BitbucketCloudCommit commit) {
         this.repository = repository;
         this.branch = branch;
         this.commit = commit;
 
         // It is possible for a PR's original destination to no longer exist.
-        if(this.branch != null) {
-            // redound available the informations into impl objects
+        if(this.branch != null && this.commit != null) {
+            // redound available the information into impl objects
```

Comment:
"redound"?

Reply:
That's from https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/152. I am gonna rephrase this to `Make the commit information available into impl objects`. cc @nfalco79

Reply:
It should was " re bind"

Reply:
@bitwiseman I want heavy remember to you [branch-api-plugin#160](https://github.com/jenkinsci/branch-api-plugin/pull/160)

Reply:
@nfalco79  I hear you.